Pear/Cells/MessagesTableViewCell.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  MessagesTableViewCell.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 5/2/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+
+import UIKit
+import Kingfisher
+
+class MessagesTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private let infoImage = UIImageView()
+    private let infoLabel = UILabel()
+    private let nameLabel = UILabel()
+    private let profileImage = UIImageView()
+    private let timeLabel = UILabel()
+
+    // MARK: - Private Data Vars
+    private let profileImageWidth: CGFloat = 50
+    private let statusData: [String: [String]] = [
+        "active": ["Current pear", "currentPear"],
```

Comment:
maybe use a named string tuple over an array since index 0 is imageName and index 1 is the status text

```
    private let statusData: [String: (image: String, status: String)] = [
        "active": (image: "Current pear", status: "currentPear"),
        "created": (image: "Current pear", status: "currentPear"),
        "proposed": (image: "Current pear", status: "currentPear"),
        "inactive": (image: "Past pear", status: "finished"),
        "canceled": (image: "Past pear", status: "finished")
    ]

```